x-pack/legacy/plugins/integrations_manager/public/constants.ts

```diff
@@ -6,39 +6,37 @@
 import { IconType } from '@elastic/eui';
 import { AssetType, ServiceName } from '../common/types';
 
+export const DisplayedAssets: Record<ServiceName, string[]> = {
+  kibana: ['index-pattern', 'visualization', 'search', 'dashboard'],
+  elasticsearch: ['index-template', 'ingest-pipeline', 'ilm-policy'],
+};
+
```

Comment:
@skh @jfsiii 
This originally looked like this:
<img width="649" alt="Screen Shot 2019-11-13 at 7 46 01 PM" src="https://user-images.githubusercontent.com/1676003/68817239-12740d80-064f-11ea-9e43-6814a3d92206.png"> .  But after merging there were type errors because AssetType is now an enum.  Trying to change the strings to the enum properties fixed the type but caused Kibana to crash for whatever reason.  If AssetType doesn't need to be an enum it would be nice to change it back to a union, unless someone can recommend a different way to improve this type.

Reply:
Interesting. In a separate branch, I also started getting strange errors like "superclass is not an object" and another one when I replaced a union with an enum. I'm still debugging because it happened just as I was closing up for the day but my code was working then I added a few statements for the enum and it crashed.

I'd like to use the enums, but I don't want to fight it too long. Perhaps we could revert and create a ticket to revisit it.

Reply:
You're getting the same results as me.  Will leave as is for now.

Reply:
> What are you thinking to revert? 

The `AssetType` `enum`

> What I have works at the moment.

Yes, but `Record<ServiceName, string[]>` loses the safety we had with `Record<ServiceName, AssetType[]>`

Reply:
Yup.  Looks like @skh is okay with us reverting so I can do that.